README.md

```diff
@@ -105,35 +102,35 @@ Data is expected in the following format:
 
-**`onError`**: callback function that is triggered when an error occurs during the Utility Connect flow.
+**`onError`**: callback function that is triggered when an error occurs during the Connect flow.
 
-**`onClose`**: callback function that is triggered when the Utility Connect is closed. The user could have closed it by clicking outside the modal or clicking a button to dismiss the modal after an error, a successful credential validation or a timeout. Provided `status` string parameter indicates the final credential submission state when Utility Component was closed. If Utility Connect was manually closed via the `close` function, the latest credential submission state will be returned. Possible states:
+**`onClose`**: callback function that is triggered when Connect is closed. The user could have closed it by clicking outside the modal or clicking a button to dismiss the modal after an error, a successful credential validation or a timeout. Provided `status` string parameter indicates the final credential submission state when Utility Component was closed. If Connect was manually closed via the `close` function, the latest credential submission state will be returned. Possible states:
 
 - `'verified'` : the credentials were confirmed to be correct
 - `'rejected'` : the credentials were confirmed to be incorrect
-- `'timed_out'`: Utility Connect times out awaiting verification of the utility credentials. In this case, if clients later receive the `UtilityCredentialRejected` webhook or confirm rejection through the UtilityCredentials API endpoint, client will likely want to redirect the user back Utility Connect (via in-app notifications or emails). If the user needs to update their credentials, the client should fetch a `UtilityConnectToken` with the pre-existing `utility_credential_id` so that Utility Connect opens in "update" mode.
-- `'pending'`: the user submitted credentials but closed Utility Connect before the component could get updated with the result
+- `'timed_out'`: Connect times out awaiting verification of the utility credentials. In this case, if clients later receive the `UtilityCredentialRejected` webhook or confirm rejection through the UtilityCredentials API endpoint, client will likely want to redirect the user back Connect (via in-app notifications or emails). If the user needs to update their credentials, the client should fetch a `UtilityConnectToken` with the pre-existing `utility_credential_id` so that Connect opens in "update" mode.
+- `'pending'`: the user submitted credentials but closed Connect before the component could get updated with the result
 - `'no_submit'`: the user never submitted their credentials
-- `'error'`: there was an API error during the Utility Connect flow and the user clicked a button to confirm as such and close Utility Connect
+- `'error'`: there was an API error during the Connect flow and the user clicked a button to confirm as such and close Connect
```

Comment:
this error description was a little confusing even before this update. Maybe we could try, "there was an API error during the Connect flow. User clicked to confirm the error and close Connect."

Reply:
```suggestion
- `'error'`: there was an API error during the Connect flow. User clicked to confirm the error and closed Connect.
```